### ISSUE TITLE
Specifier ordering different in version 0.4?

### ISSUE THREAD

**Issue by krisztianb** [KyleMayes/eslint-plugin-ordered-imports#13]
So far I was using version 0.3.1 of this plugin with the following config:

```json
"ordered-imports/ordered-imports": [
  "error",
  {
    "declaration-ordering": ["source", "case-insensitive"],
    "specifier-ordering": "case-insensitive"
  }
],
```

It showed no error for the following import:

    import { a, b, _ } from "someModule";

After updating to 0.4.0 it wants the import to look like this:

    import { _, a, b } from "someModule";

For me this is undesirable, because it is not how VSCode/TS sorts imports with its "Organize Imports" (keyboard shortcut: SHIFT + ALT + O) feature. Now everytime I organize the imports in VSCode, I have to manually change these import statements.

Any ideas why this was changed and a possible workaround?

**Comment by krisztianb** [comment]
Found the culprit commit here: https://github.com/KyleMayes/eslint-plugin-ordered-imports/commit/268aeee5d9cd27970c675363ec791f5aa7951020

Any chance to make this optional?

**Comment by KyleMayes** [comment]
Released `v0.4.1` which adds the `"symbols-first"` option which, when set to `false`, will cause the plugin to use the default string sorting that was in use before `v0.4.0`. Definitely should have added this as part of `v0.4.0`, not sure why I didn't think about it.

**Comment by krisztianb** [comment]
Thank you very much.
